### PR TITLE
*fix Improve table columns resizing

### PR DESCRIPTION
**Pull request by Sodik**
*feat Table context menu

### CODE REVIEW

File: packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react'
```
Comment (allfayn) [inline packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx:1]: Should we add e2e tests for the new component?

Comment (Sodik) [reply packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx:1]: We should. I just don't know when we could setup it

Comment (allfayn) [reply packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx:1]: I talking about screenshot/loki tests, that we have already.

Comment (Sodik) [reply packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx:1]: done

**Review by utkukaratas — APPROVED** [review]

**Pull request merged**